configure.ac

```diff
@@ -167,6 +167,13 @@ AC_ARG_ENABLE([zmq],
 
 AC_ARG_WITH([protoc-bindir],[AS_HELP_STRING([--with-protoc-bindir=BIN_DIR],[specify protoc bin path])], [protoc_bin_path=$withval], [])
 
+# Enable manpages
+AC_ARG_ENABLE(man,
+    [AS_HELP_STRING([--enable-man],
```

Comment:
Normally we use --disable-* for the help for options which are enabled by default.